lib/storage/index.ts

```diff
@@ -22,112 +61,121 @@ const Storage: Storage = {
      * and enables fallback providers if necessary
      */
     init() {
-        provider.init();
+        tryOrDegradePerformance(() => {
+            provider.init();
+        }).finally(() => {
```

Comment:
```suggestion
        tryOrDegradePerformance(provider.init).finally(() => {
```
Is it OK to simplify this?